src/containerapp/HISTORY.rst

```diff
@@ -6,6 +6,11 @@ upcoming
 ++++++
 * 'az containerapp up/create/update': Wait longer time for logstream of Cloud Build to make sure the container start
 
+0.3.51
```

Comment:
maybe we need to add the history into the "upcoming" part.

Reply:
moved to "upcoming" part